src/classes/BGE_FormTemplate_TDTM.cls

```diff
@@ -46,39 +46,56 @@ public with sharing class BGE_FormTemplate_TDTM extends TDTM_Runnable{
     ********************************************************************************************************/
     public override DmlWrapper run(List<SObject> newlist, List<SObject> oldlist,
             TDTM_Runnable.Action triggerAction, Schema.DescribeSObjectResult objResult) {
-        validateFormTemplateDeletion(newlist, oldlist, triggerAction);
+
+        if (triggerAction == TDTM_Runnable.Action.BeforeDelete) {
+            validateFormTemplateDeletion( oldlist );
+        }
+
         return null;
     }
 
 
     /*******************************************************************************************************
      * @description Validates that a Form Template Record is not used by any Data Import Batch record
-     * (If it is, an error is added to the Form Template record)
-     * @param newlist the list of Form Templates from trigger new.
+     *              or defined as default on gift_entry custom setting
+     *              (If it is, an error is added to the Form Template record)
      * @param oldlist the list of Form Templates from trigger old.
-     * @param triggerAction which trigger event (BeforeInsert, AfterInsert, etc.)
      *******************************************************************************************************/
-    private void validateFormTemplateDeletion (List<SObject> newlist, List<SObject> oldlist,
-            TDTM_Runnable.Action triggerAction){
+    private void validateFormTemplateDeletion ( List<SObject> oldlist ){
 
-        if (triggerAction == TDTM_Runnable.Action.BeforeDelete) {
-            List<Form_Template__c> formTemplates = (List<Form_Template__c>) oldlist;
+        // aux vars
+        List<Form_Template__c>      formTemplates           = (List<Form_Template__c>) oldlist;
```

Comment:
Let's remove the extra spaces around the '=', to be consistent with the npsp code formatting.

Reply:
I was trying to improve variable legibility (check img below) but i'm going to do as you said and keep consistency with rest of the code.

![Screen Shot 2020-01-24 at 10 38 01 AM](https://user-images.githubusercontent.com/3368871/73094567-ac4d0780-3e95-11ea-90ef-7c10290684e4.png)